packages/password-input/src/TogglePassword/TogglePassword.tsx

```diff
@@ -20,8 +19,6 @@ export const TogglePassword = ({
     <>
       <IconButton
         aria-label="Toggle password visibility"
-        aria-disabled={disabled}
-        disabled={disabled}
         onClick={handleTogglePasswordClick}
         className={cx(baseStyles, sizeStyles[size])}
```

Comment:
Check with design, but I wonder if the button should get darker when the input is disabled. The button looks disabled even though it isn't